### PR TITLE
table pdf icons update

### PR DESCRIPTION
**Pull request by kialj876**
Signed-off-by: Kial Jinnah <kialj876@gmail.com>

*Issue #:* /bcgov/entity#9111

*Description of changes:*
- see ticket for details

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]

File: ppr-ui/src/components/tables/SearchHistory.vue [hunk 1]
```diff
@@ -25,13 +25,83 @@
...
-              <v-icon class="ma-0" left small>mdi-file-pdf-outline</v-icon>
-              <span :class="[$style['pdf-btn-text'], 'ma-0']">PDF</span>
+            <v-btn
+              :id="`pdf-btn-${item.searchId}`"
+              class="pdf-btn px-0 mt-n3"
+              depressed
+              :loading="item.searchId === loadingPDF"
+              @click="downloadPDF(item.searchId)"
+            >
+              <svg
```
Comment (jordiwes) [inline ppr-ui/src/components/tables/SearchHistory.vue]: Do you not just add the img svg at this point?
Alterations example:
``

Comment (kialj876) [reply ppr-ui/src/components/tables/SearchHistory.vue]: Oh interesting, I didn't know this was a thing. I will try that out

Comment (kialj876) [reply ppr-ui/src/components/tables/SearchHistory.vue]: Oh yeah that's much nicer haha

**Review by jordiwes — APPROVED** [review]
sweet

**Pull request merged**